### DISCUSSION TITLE
進捗や優先度の高いものがわかりやすいようにProjectsを作った

### DISCUSSION THREAD

**Discussion by pss-aileen** [first-contributions-ja/first-contributions-ja.github.io#26] [【運営】相談・報告]
![スクリーンショット 2024-03-13 9 24 31](https://github.com/first-contributions-ja/first-contributions-ja/assets/140990100/82ccfb7c-e752-423c-8d7b-0dc5b70e139f)

[運営にあたっての基盤作り](https://github.com/orgs/first-contributions-ja/projects/1)

勝手に作ってみました。

優先度を確認できるので良いのかな...どうかな...と思ったりです。

**Comment by kagomen** [comment]
Projects使ってみたかったのでありがたいです！
現在[first-contributions-ja](https://github.com/first-contributions-ja/first-contributions-ja)のProjectsページが空の状態ですが、編集中ですかね❔

**Reply by pss-aileen** [reply]
招待しないとみれない感じでした！！！
今とりあえず @kagomen @kazzyfrog さん招待しました！
ご確認ください！

**Reply by kagomen** [reply]
ありがとうございます🙏
